tt_eager/tt_dnn/op_library/sliding_window_op_infra/tt_py_composite_conv.py

```diff
@@ -366,9 +340,6 @@ class TTPyCompositeConv(TTPyOp):
         "per_core_weight_matrix_width",
         "act_block_h",
         "act_block_w",
-        "act_c_num_blocks",
-        "weight_block_w",
-        "out_block_h",
         "out_block_w",
```

Comment:
do we need "out_block_w" or can it be derived?

Reply:
Looks like "out_block_w" is unused and not set at all. We can probably delete it from config_keys in another check-in.

Reply:
> Looks like "out_block_w" is unused and not set at all. We can probably delete it from config_keys in another check-in.

Sure. Noted.